src/chapters/thymeleaf-views/template-conditionals.rst

```diff
@@ -0,0 +1,237 @@
+Conditionals in a Template
+===========================
+
+In addition to iteration, Thymeleaf can also add or remove content on a
+webpage based on certain conditions. Going back to the coffee example, we could
+generate the ordered list ONLY IF ``coffeeOptions`` contains data. If the
+ArrayList is empty, then there is no need to include the ``<ol>`` element.
+Instead, the template could include a ``<p>`` element with text stating that
+there are no options to select.
+
+Just like the ``for/each`` syntax differs between Java and Thymeleaf, we need
+to examine how to include *conditionals* in our templates. The logic remains
+the same, but the implementation requires practice.
+
+Try It!
+--------
+
+Code along with the following video to gain more practice with Thymeleaf:
+
+   Video goes here...
+
+For your reference, here is an extension for some of the concepts presented
+in the clip. Note that the text is NOT intended as a replacement for the
+walkthrough. To get better at coding, you need to actually CODE instead of just
+reading about how to do it.
+
+Display Content ``if``
+-----------------------
+
+The general syntax for including a conditional in Thymeleaf is:
+
+.. sourcecode:: groovy
+
+   th:if = "${condition}"
+
+#. The ``th:if`` statement gets placed inside an HTML tag.
+#. ``condition`` represents a boolean variable provided by the controller.
+#. Alternatively, ``condition`` can be a statement that evaluates to ``true``
+   or ``false``.
+
+.. admonition:: Note
```

Comment:
It also might not look like the above. I would remove this note.

Reply:
Done.  Thanks.